service/qos/service_level_controller.cc

```diff
@@ -1180,7 +1180,9 @@ future<std::vector<cql3::description>> service_level_controller::auth_integratio
 }
 
 future<std::vector<cql3::description>> service_level_controller::describe_service_levels() {
-    SCYLLA_ASSERT(_auth_integration != nullptr);
+    if (_auth_integration == nullptr) {
+        throw std::runtime_error("auth::service has not been registerd for the service level controller yet");
+    }
```

Comment:
We expect to throw this `runtime_error` only for maintenance socket, so perhaps we should help the user understand that they are probably doing something unsupported. The current error message suggests a bug once the node fully restarts in maintenance mode.